internal/neofs/neofs.go

```diff
@@ -575,3 +587,25 @@ func (x *AuthmateNeoFS) CreateObject(ctx context.Context, prm tokens.PrmObjectCr
 		Payload: bytes.NewReader(prm.Payload),
 	})
 }
+
+func isHomomorphicHashDisabled(ctx context.Context, p *pool.Pool) (res bool, err error) {
+	ni, err := p.NetworkInfo(ctx)
+	if err != nil {
+		return false, err
+	}
+
+	var (
+		expectedParamKey   = []byte("HomomorphicHashingDisabled")
+		expectedParamValue = []byte{1}
+	)
+
+	ni.NetworkConfig().IterateParameters(func(p *netmap.NetworkParameter) bool {
```

Comment:
You can also see how it's implemented in SDK branch https://github.com/nspcc-dev/neofs-sdk-go/blob/be747ff11edd6c59a47d3b5d2b221e5754ac11e1/container/network.go#L51.
Note that there are still no PR in NeoFS API with attribute format, but I guess we'll require `true` string.